src/facade/conn_context.h

```diff
@@ -44,12 +45,19 @@ class ConnectionContext {
   }
 
   // connection state / properties.
-  bool async_dispatch: 1;  // whether this connection is currently handled by dispatch fiber.
-  bool conn_closing: 1;
-  bool req_auth: 1;
-  bool replica_conn: 1;
-  bool authenticated: 1;
-  bool force_dispatch: 1;   // whether we should route all requests to the dispatch fiber.
+  bool async_dispatch : 1;  // whether this connection is currently handled by dispatch fiber.
+  bool conn_closing : 1;
+  bool req_auth : 1;
+  bool replica_conn : 1;
+  bool authenticated : 1;
+  bool force_dispatch : 1;  // whether we should route all requests to the dispatch fiber.
+  bool monitor : 1;  // when a monitor command is sent over a given connection, we need to aware of
```

Comment:
nit: you do not use it in facade code, and in server code you have `connection_state.monitor` that should suffice.

Reply:
moved